src/parser/jobs/brd/modules/AdditionalStats.js

```diff
@@ -8,19 +8,15 @@ import STATUSES from 'data/STATUSES'
 import math from 'mathjsCustom'
 
 // Relevant crit buffs
+// TODO: Add DNC crit buffs when the statues gets added
```

Comment:
This is more a comment for the file overall, but - aren't crits no longer a factor for BRD procs? Unless they changed it back, I'm pretty sure Repertoire stacks build on flat DoT ticks now, not crits specifically. It's probably outside the scope of this PR (especially if we want to keep them small), but you can probably cut this file down a lot to remove all the wacky math for determining crit rate.

Reply:
This is true, I have avoided changing anything there because I had no reason too and I didn't want to mess it up without fully understanding it's purpose first.  I'll handle this in a different pr though.